week-3.md

```diff
@@ -0,0 +1,82 @@
+ * PoC work: None! (33), 3+ hours (1), 1 hour (1), 30 minutes (1)
+
+Most people rated their first week progress as 2 or 3, so about as expected, and resources as 3-5 which is great. Study group rating was bit more spread out, with nine responses give a low rating of 1 or 2, twenty giving mid-good rating of 3 or 4, and six giving top marks of 5. So if your group is doing particularly well, it might be worth writing a blog post or making a youtube video or similar about what's going right so others can learn from you; alternatively if your group is not doing so great, it might be interesting to lurk through some of the other groups' slack channels to see what works for them and perhaps try it in your group.
+
+This poll was just for the first week, so it's already out of date, but since this is all an experiment, might be of interest anyway.
+
+## Results from Week Two
+
+We've had the second two Q&A sessions, with meetbot logs: [week 2 Q&A 1](), [week 2 Q&A 2]() -- but as it turns out the second Q&A didn't actually have any questions. Since Q&A 2 in week one seemed a bit sparser too, we'll just advertise the single slot this week, but if that's not convenient for you, do feel free to ask questions anytime -- experts are lurking in that channel most of the time, and even if they're not available to answer immediately, if you can stay connected you will often get an answer sometime later. (If you can't stay connected, you can use the [logs](http://www.erisian.com.au/taproot-bip-review/) to catch your answer, but it might be wise to mention that you'll be doing so to avoid people thinking there's no point responding because you've left the channel)
+
+There's been a few more suggested improvements for the BIPs, including [#138](https://github.com/sipa/bips/issues/138), [PR#145](https://github.com/sipa/bips/pull/145), [PR#144](https://github.com/sipa/bips/pull/144), [PR#137](https://github.com/sipa/bips/pull/137), and [PR#136](https://github.com/sipa/bips/pull/136). Max Hillebrand has filed an [issue for high-level discussion of adding taproot to Wasabi Wallet](https://github.com/zkSNACKs/WalletWasabi/issues/2531) that might be interesting to see.
+
+> "that may have been too obvious for me when writing it" -- Pieter Wuille, [personal correspondence 2019-11-13](#correspondence)
+
+One thing that might not be obvious is that after working on something complicated and technical like this, it's possible to forget which things are obvious to everyone and which things are only obvious after you've been studying the subject matter deeply. The quote above is in regard to the bech32 issue discussed last week and that, and how something that seems like a surprising bug to some might just be a completely expected trade off to someone more deeply steeped in the technology. Hopefully wide review amongst people with varying levels of knowledge and experience can help reduce these sorts of misunderstandings.
```

Comment:
Is this an appropriate link for "bech32 issue discussed" text?
http://www.erisian.com.au/taproot-bip-review/log-2019-11-05.html#l-231

Reply:
dropped this paragraph